modules/seqexec/web/client/src/main/scala/seqexec/web/client/model/ModelOps.scala

```diff
@@ -115,6 +120,14 @@ object ModelOps {
     }
   }
 
+  implicit class CamelCaseOps(val s: String) extends AnyVal {
+    def toCamelCase: String =
+      (s.toList match {
+        case Nil       => Nil
+        case x :: rest => x.toUpper :: rest.map(_.toLower)
+      }).mkString
```

Comment:
I'm not 100% clear how this works with camel case. Isn't it just capitalizing the first character and making everything else lowercase?

Reply:
Right, maybe this shouldn't be called camelCase. What name would it be more appropriate?

Reply:
I was going to say the same, but Sebastian beat me.

Reply:
'PhraseCase' ?

Reply:
I think it's generally called TitleCase.

Reply:
i  looked around and settled on `Sentence case`. There are multiple versions but it seems `Title Case` will imply more than one capitalized letter and I only wanted the first one capitalized